app/lib/gateway/optics.rb

```diff
@@ -0,0 +1,41 @@
+module Gateway
+  class Optics
+    class ClientError < StandardError; end
+
+    GET_TOKEN_ENDPOINT = 'https://uat.icasework.com/token?db=hmcts'.freeze
+    ENDPOINT = 'https://uat.icasework.com/createcase?db=hmcts'.freeze
```

Comment:
not needed for this PR but make these ENV vars

Reply:
Yeah, will pull these out